### DISCUSSION TITLE
求更新钢铁雄心4 1.10.*版本

### DISCUSSION THREAD

**Discussion by floodnice** [Gamuxorg/bbs#279] [投递和请求游戏]
游戏名称：钢铁雄心4
游戏名称(英文)：
下载地址 :（如果有的话）：

**Comment by wangling12** [comment]
已经更新至原页面https://www.linuxgame.cn/%e9%92%a2%e9%93%81%e9%9b%84%e5%bf%834-hearts-of-iron-iv
